ci/docker_nlt.sh

```diff
@@ -10,6 +10,7 @@ set -x
 
 . utils/sl/setup_local.sh
 
+ps auwx
```

Comment:
for debugging purpose?

Reply:
right to debug a NLT failure

Reply:
Now fixed in https://github.com/daos-stack/daos/pull/13726